packages/components/src/entities/SampleFinderSection.tsx

```diff
@@ -278,6 +295,13 @@ const SampleFinderSectionImpl: FC<Props & InjectedAssayModel> = memo(props => {
             onFilterClose();
             updateFilters(filterChangeCounter + 1, newFilterCards, !currentView?.entityId, true);
 
+            if (isAssay) {
+                api.query.incrementClientSideMetricCount(
+                    SAMPLE_FILTER_METRIC_AREA,
+                    'with ' + assayDesignCount + ' AssayDesgin' + (assayDesignCount > 1 ? 's' : '')
```

Comment:
it is odd to have a metric name with spaces in it. I don't think we do that elsewhere. Also odd to have the metric name be essentially the metric. But I can't think of a different way to do this assay design count via the client side metrics